UITests/Tests/AxeScanAllTests.cs

```diff
@@ -0,0 +1,124 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using OpenQA.Selenium.Appium.Windows;
+using System;
+using System.Linq;
+using System.Text.Json;
+using System.IO;
+using System.Collections.ObjectModel;
+
+namespace UITests.Tests
+{
+    [TestClass]
+    public class AxeScanAll : TestBase
+    {       
+        public static readonly string jsonUri = "ms-appx:///../../../../../../WinUIGallery/DataModel/ControlInfoData.json";
+        public static new WindowsDriver<WindowsElement> Session => SessionManager.Session;
+
+        public class ControlInfoData
+        {
+            public ObservableCollection<ControlInfoDataGroup> Groups { get; set; }
+        }
+
+        public class ControlInfoDataGroup
```

Comment:
I'm curious what you think about this data duplication, should these classes exist in a shared project and be referenced by both the Gallery and this project?